Ryujinx.HLE/HOS/ModLoader.cs

```diff
@@ -381,66 +382,82 @@ internal bool ReplaceExefsPartition(ulong titleId, ref IFileSystem exefs)
             return true;
         }
 
-        internal bool ApplyExefsMods(ulong titleId, List<NsoExecutable> nsos)
+        public struct ModLoadResult
         {
+            public BitVector32 Stubs;
+            public BitVector32 Replaces;
+            public Npdm Npdm;
+
+            public bool Modified => (Stubs.Data | Replaces.Data) != 0;
+        }
+
+        internal ModLoadResult ApplyExefsMods(ulong titleId, NsoExecutable[] nsos)
+        {
+            ModLoadResult res = new ModLoadResult();
```

Comment:
You can set the fields like that. And probably rename the var to something a bit more explicit.
```suggestion
            ModLoadResult modLoadResult = new ModLoadResult
            {
                Stubs    = new BitVector32(),
                Replaces = new BitVector32()
            };
```